### PR TITLE
Portfolio mobile version

### PR DESCRIPTION
**Pull request by mohashyne**
- [ ]  Add Html file
- [ ]  Add CSS file
- [ ]  Add HTML & CSS linters
- [ ]  Edit ReadMe file
- [ ]  Add MIT License File
- [ ]  Add Header & Hero Component
- [ ]  Add styling
- [ ]  Add images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,186 @@
...
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+**This is the portfolio for Muhammad Aminu Salihu as a Software Developer**
```
Comment (franclobo) [inline README.md:38]: - [ ] Please 🙏🏽 add the title in the main header, your Readme should look like the template for professional documentation 👇🏽 

image

File: README.md [hunk 2]
```diff
@@ -1 +1,186 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> comming soon.
+
+- [Live Demo Link](https://google.com)
```
Comment (franclobo) [inline README.md:71]: - [ ] Kindly remove the live demo section until you deploy the app for professional documentation. 👍🏽

File: README.md [hunk 3]
```diff
@@ -1 +1,186 @@
...
+
+### Run tests
+
+comming soon
+
+### Deployment
+
+You can deploy this project using:
+
+comming soon
```
Comment (franclobo) [inline README.md:127]: - [ ] Similar to the live demo section, please 🙏🏽 remove this until you have done the deployment or tests for your app for professional documentation.

**Review by franclobo — COMMENTED** [review]
Hi @mohashyne, 

terminator

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Gambit142 — APPROVED** [review]
Hi @mohashyne 👋,

## Status: Approved 🟢   

Your project is complete!  You did a great job implementing features for this milestone 🥳👏. There is nothing else to say other than... it's time to merge it :shipit: 

#### Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Use of semantic HTML ✔️ 
- Commit messages are capitalized and in imperative tense ✔️ 
- Good implementation of project's requirements ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**